Utilities/ClangFormat/DownloadClangFormat.cmake

```diff
@@ -43,6 +43,8 @@ endif()
 if(ITK_FORBID_DOWNLOADS)
   message(SEND_ERROR "Attempted to download ClangFormat when ITK_FORBID_DOWNLOADS is ON")
 endif()
+cmake_policy(PUSH)
+cmake_policy(SET CMP0135 NEW)
```

Comment:
This line triggers a CMake warning for all versions of CMake that do not know CMP0135 like the minimum required versions of ITK, for example. A policy check like `if(POLICY CMP0135)` is missing.